pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -309,35 +307,28 @@ pub mod pallet {
 		/// Sets the router IDs used for a specific domain,
 		#[pallet::weight(T::WeightInfo::set_domain_routers())]
 		#[pallet::call_index(0)]
-		pub fn set_domain_routers(
+		pub fn set_routers(
 			origin: OriginFor<T>,
-			domain: Domain,
 			router_ids: BoundedVec<T::RouterId, T::MaxRouterCount>,
 		) -> DispatchResult {
 			T::AdminOrigin::ensure_origin(origin)?;
 
-			ensure!(domain != Domain::Centrifuge, Error::<T>::DomainNotSupported);
-
-			//TODO(cdamian): Outbound -  Call router.init() for each router?
+			<Routers<T>>::set(router_ids.clone());
```

Comment:
Still a bit confused by this setup since the routers that we're storing at not the ones that we are using when processing...

Anyway, here I'm storing all the available routers right @lemunozm?

Reply:
So if a message comes from a not configured router, then the gateway will discard it